app/models/satisfaction_survey_feature_metrics.rb

```diff
@@ -42,6 +60,16 @@ def application_forms_with_feedback(start_time, end_time)
       .where(recruitment_cycle_year: RecruitmentCycle.current_year)
   end
 
+  def application_forms_with_positive_feedback(start_time, end_time)
+    ApplicationForm
+      .where(
+        recruitment_cycle_year: RecruitmentCycle.current_year,
+      )
+      .where('application_forms.submitted_at BETWEEN ? AND ?', start_time, end_time)
```

Comment:
I think this should work?:

```suggestion
      .where(application_forms: {submitted_at: start_time..end_time})
```

Reply:
will give it a go, was cribbing from Steve's sql